ui/src/app/utils/access-tiers.tsx

```diff
@@ -6,6 +6,11 @@ export enum AccessTierShortNames {
   Controlled = 'controlled',
 }
 
+export const orderedAccessTierShortNames = [
```

Comment:
I believe we have a structure like this in a few other places.  We should change those to references to here, in a future update.

Reply:
Done in [6269](https://github.com/all-of-us/workbench/pull/6269)